dataminrpulse.json

```diff
@@ -0,0 +1,1978 @@
+                        1234563
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.name",
+                    "data_type": "string",
+                    "example_values": [
+                        "watchlist_name"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.properties.watchlistColor",
+                    "data_type": "string",
+                    "example_values": [
+                        "purple"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.type",
+                    "data_type": "string",
+                    "example_values": [
+                        "COMPANY"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string",
+                    "example_values": [
+                        "1"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.total_results",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string",
+                    "example_values": [
+                        "Total watchlists: 1"
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "custom",
+                "width": 4,
+                "height": 1,
+                "view": "dataminrpulse_view.display_alerts",
+                "title": "Watchlist Details"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "get alerts",
+            "identifier": "get_alerts",
+            "description": "Fetch the details of the alerts from the Dataminr platform for the given List ID or query",
+            "type": "investigate",
+            "read_only": true,
+            "parameters": {
+                "list_id": {
+                    "description": "List ID",
+                    "data_type": "string",
+                    "primary": true,
+                    "contains": [
+                        "dataminrpulse list id"
+                    ],
+                    "order": 0
```

Comment:
`order` = `1` is missing in this list of parameters

Reply:
Done